tests/common/playground.rs

```diff
@@ -413,26 +426,28 @@ impl CBPlayground {
         username: String,
         password: String,
     ) -> String {
-        let uri = format!(
-            "{}/pools/default/buckets/{}/scopes/{}/collections",
-            conn_string, bucket, scope
-        );
-
         let mut uuid = Uuid::new_v4().to_string();
         uuid.truncate(6);
         let collection_name = format!("test-{}", uuid);
 
         let mut params = HashMap::new();
         params.insert("name", collection_name.clone());
 
-        let client = reqwest::Client::new();
-        let res = client
-            .post(uri)
-            .form(&params)
-            .basic_auth(username, Some(password))
-            .send()
-            .await
-            .unwrap();
+        let uri = build_uri(conn_string.clone(), bucket, scope).await;
+
+        let res = task::spawn_blocking(move || {
```

Comment:
I'm pretty sure that reqwest has a blocking API doesn't it?

Reply:
Resolving as have discussed on slack